release/golden_notebook_tests/workloads/torch_tune_serve_test.py

```diff
@@ -186,7 +189,8 @@ def test_predictions(test_mode=False):
 
     print("Retrieving best model.")
     best_checkpoint = analysis.best_checkpoint
-    model_id = get_best_model(best_checkpoint)
+    get_best_model_remote = ray.remote(get_best_model)
```

Comment:
Is it possible to test out the `ray.client().download_results()` workflow instead of wrapping in a task?